frontends/infinite-corridor/src/components/LearningResourceCard.tsx

```diff
@@ -40,10 +42,11 @@ const LearningResourceCard: React.FC<LearningResourceCardProps> = ({
         footerActionSlot={
           user.is_authenticated && (
             <IconButton
+              size="small"
               aria-label="Add to list"
               onClick={() => addToList.open(resource)}
             >
-              <BookmarkBorderIcon />
+              {isInList ? <BookmarkIcon /> : <BookmarkBorderIcon />}
```

Comment:
Under `/learn/`, if I add a resource to a list, the bookmark is filled in right away, but under `/infinite` I have to reload the page or redo a search before the bookmark is filled in.

Reply:
fine to fix in a future PR

Reply:
https://github.com/mitodl/open-discussions/issues/3908